.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,16 @@
+name: PR Image Docker Cleanup
+on:
+  pull_request:
+    types: [closed]
```

Comment:
does this need to be further filtered to only those issues that had an image built as a result of the `build-docker-image` workflow?

Reply:
Ideally yes..  but how?  Also, there isn't much downside to running this on PRs that didn't generate an image.